src/server/models/db/prisma-client.ts

```diff
@@ -63,6 +63,6 @@ prisma.$on("error", (e) => {
   logger.error(e);
 });
 
-prisma.$on("beforeExit", () => {
+process.on("beforeExit", () => {
```

Comment:
New Prisma doesn't like it 
```sh
Error: "beforeExit" hook is not applicable to the library engine since Prisma 5.0.0, it is only relevant and implemented for the binary engine. Please add your event listener to the `process` object directly instead.
```